python/test/unit/language/test_core.py

```diff
@@ -5482,6 +5484,77 @@ def test_local_load_store(M, N, K, dist_layout, shared_layout, device, tmp_path:
     assert torch.equal(z, x)
 
 
+mma_layout = [
```

Comment:
Nit. Any chance you can sprinkle a couple `DotOperand`s here?

Reply:
We don't use `stmatrix` for `DotOperandLayout` for now

Reply:
Of course I think the precondition can be removed for arbitrary layouts as long as they satisfy certain conditions.

Reply:
Let me add a check to assert that `stmatrix` appear in ptx